.github/CODEOWNERS

```diff
@@ -16,4 +16,4 @@
 *   @open-telemetry/specs-approvers
 
 # Owners for the semantic conventions generator tool
-/semantic-conventions/      @open-telemetry/specs-approvers @Oberon00
```

Comment:
Removing @Oberon00's explicit mention since he's already part of @open-telemetry/specs-semconv-approvers 🙂